README.md

```diff
@@ -129,14 +129,18 @@ If you have either created your own build of the MailOnline plugin or are replac
 
 #### Using your own rendering code
 
-If you are replacing the MailOnline rendering plugin with your own custom rendering code, you need to change the code where currently the MailOnline plugin is loaded and invoked.
+If you are replacing the MailOnline rendering plugin with your own custom rendering code, you need to change the code where currently an ad renderer is loaded and invoked.
 
+- You have to explicitly set `adRendere` options to `custom` value.

```

Comment:
typo: `adRendere` >> should be: `adRenderer`.
Also, I would reword it to read like this:
You have to explicitly add the  `adRenderer` param to your configuration object with this value: `adRenderer: "custom"`.